src/components/Button/Button.tsx

```diff
@@ -62,6 +72,10 @@ export const Button: React.SFC<ButtonProps & React.ButtonHTMLAttributes<HTMLButt
     </button>
   );
 };
+
```

Comment:
These statements don't need whitespace around them (and similar whitespace isn't applied elsewhere)